json-format.md

```diff
@@ -92,9 +92,12 @@ respective CloudEvents type when the mapping rules are fulfilled.
 ### 2.3. Mapping Data
 
 If an implementation determines that the actual type of `data` is a `String`,
-the value MUST be represented as [JSON string][json-string] expression; for
-`Binary`, the value MUST represented as [JSON string][json-string] expression
-containing the [Base64][base64] encoded binary value.
+the value MUST be represented as [JSON string][json-string] expression and the
+member name inside the JSON object MUST be `data`. For `Binary`, the value MUST
+be represented as a [JSON string][json-string] expression containing the 
+[Base64][base64] encoded binary value, and the member name inside the JSON 
+object MUST be `data_base64`.
```

Comment:
While it's implied, it doesn't actually ban someone from having both `data` and `data_base64` in the JSON. Can we add something like:
```
Note, `data` and `data_base64` MUST NOT both appear at the same time within a CloudEvent serialization.
```
Someone may try to be "smart" and fill in both (when possible, like when it's a string) and that's a situation I think we need to avoid.

Also, could someone misinterpret the `Binary` usage here? I believe you mean that the data type of `data` is binary and you're not referring to the binary serialization of the CE.  Perhaps:
- - -
For cases where `data` is a `Binary` value, it MUST be represented ....
- - -